prdoc/schema_user.json

```diff
@@ -0,0 +1,212 @@
+        "items": {
+          "$ref": "#/$defs/doc"
+        },
+        "minItems": 1
+      },
+
+      "crates": {
+        "title": "Crates",
+        "description": "You have the option to provide a hint about the crates that have noticeable changes.\n This is used during the crate publishing to crates.io and to help users understand the impact of the changes introduced in your PR.",
+        "type": "array",
+        "items": {
+          "$ref": "#/$defs/crate"
+        }
+      },
+
+      "migrations": {
+        "title": "Migrations (DB & Runtime)",
+        "description": "It is important for users to be aware of migrations.\nMake sure to mention any migrations in the appropriate sub-properties:\n- db\n- runtime",
+        "type": "object",
+        "properties": {
+          "db": {
+            "type": "array",
+            "nullable": false,
+            "title": "Database Migration",
+            "description": "List of the Database Migrations or empty array: []",
+            "items": {
+              "$ref": "#/$defs/migration_db"
+            },
+            "minItems": 0,
+            "required": [
+              "name",
+              "description"
+            ]
+          },
+          "runtime": {
+            "type": "array",
+            "title": "Runtime Migration",
+            "nullable": false,
+            "description": "List of the Runtime Migrations or empty array: []",
+            "minItems": 0,
+            "items": {
+              "$ref": "#/$defs/migration_runtime"
+            },
+            "required": [
+              "db",
+              "runtime"
+            ]
+          }
+        }
+      },
+      "host_functions": {
+        "title": "Host Functions",
+        "description": "List of the host functions involved in this PR.",
+        "type": "array",
+        "items": {
+          "$ref": "#/$defs/host_function"
+        }
+      }
+    },
+    "required": [
+      "title",
+      "doc",
+      "crates"
+    ],
+    "$defs": {
+      "audience": {
+        "description": "You may pick one or more audiences and address those users with appropriate documentation, information and warning related to the PR.",
+        "oneOf": [
+          {"const": "Node Dev",
+          "title": "Node Dev",
+          "description": "Those who build around the client side code. Alternative client builders, SMOLDOT, those who consume RPCs. These are people who are oblivious to the runtime changes. They only care about the meta-protocol, not the protocol itself."},
+
+          {"const": "Runtime Dev",
+          "title": "Runtime Dev",
+          "description": "All of those who rely on the runtime. A parachain team that is using a pallet. A DApp that is using a pallet. These are people who care about the protocol (WASM), not the meta-protocol (client)."},
+
+          {"const": "Node Operator",
+          "title": "Node Operator",
+          "description": "Those who don't write any code and only run code."},
```

Comment:
Is it intended that there is no "Validator" option? We would just choose "Node Operator" for that case?

Reply:
Yes. Most of the changes are equally for both groups and normal node operators are also able to just skip the one validator message for every 5th release.